qiskit_optimization/algorithms/grover_optimizer.py

```diff
@@ -82,51 +86,35 @@ def __init__(
         if quantum_instance is not None and sampler is not None:
             raise ValueError("Only one of quantum_instance or sampler can be passed, not both!")
 
-        self._quantum_instance = None  # type: Optional[QuantumInstance]
-        if quantum_instance is not None:
-            warnings.warn(
-                "The quantum_instance argument has been superseded by the sampler argument. "
-                "This argument will be deprecated in a future release and subsequently "
-                "removed after that.",
-                category=PendingDeprecationWarning,
-                stacklevel=2,
+        if quantum_instance:
+            warn_deprecated(
```

Comment:
Since this is now deprecated, while it was noted in the prior release that it was pending deprecation and would be deprecated in the future, now that it its being done we should have a release note with a deprecations section that notes this.

Reply:
Thanks. I will do it.

Reply:
done